src/seaport.ts

```diff
@@ -2927,10 +2961,80 @@ export class OpenSeaPort {
       ]
     ]
 
+    const useFeeWrapper = isFeeWrapperFlow({ buy, sell }, this._networkName)
+
+    let wyvernFeeWrapperArgs: WyvernFeeWrapperAtomicMatchParameters | undefined
+
+    if (useFeeWrapper && makerOrder?.hash) {
+      const { fulfillmentData } = await this.api.getOrderFulfillmentData(makerOrder.hash)
+
+      wyvernFeeWrapperArgs = [args, fulfillmentData.serverSignature || '', fulfillmentData.feeData]
```

Comment:
Should we throw exception/fail here is serverSignature is empty? I think that's weird

Reply:
Also for testing, wanna log if feeData is empty?

Reply:
the backend should verify that feeData is empty though, is that what you meant?